.github/workflows/ci.yml

```
@@ -668,4 +673,3 @@ jobs:
           files: |
             ./**/*.whl
             ./**/*.tar.gz
```

```suggestion
            ./**/*.tar.gz
            ./**/*-wheelhouse-*.zip
```
Wheelhouse is still needed in GitHub release